tests/cupy_tests/core_tests/test_raw.py

```diff
@@ -1143,6 +1148,9 @@ def test_grid_sync_rawmodule(self):
     'compile': (False, True),
     'raw': ('ker', 'mod', 'mod_ker'),
 }))
+@unittest.skipUnless(
+    60 <= int(cupy.cuda.device.get_compute_capability()),
+    'Requires compute capability 6.0 or later')
```

Comment:
Question: what does CC have to do with pickling?

Reply:
This is because `enable_cooperative_groups=True` is used in tests. Maybe we can instead use `translate_cucomplex` attribute so that we don't have to care about CC.

Reply:
Ah, that's a good point! We can mark `translate_cucomplex` as `readonly` in `raw.pxd`, then we can replace `enable_cooperative_groups` by it. (The attribute being exposed to Python was the main reason I didn't use it in the first place 😅)